libs/design/src/core/colorable/colorable.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { daffColorMixin } from "./colorable";
+import { ElementRef } from "@angular/core";
+
+/**
+ * Some of these tests are inspired by the @angular/material
+ * color common-behavior, as it is a really clean implementation.
+ */
+describe('daffColorMixin', () => {
+    let instance;
+    let classWithColor;
+
+    beforeEach(() => {
+        classWithColor = daffColorMixin(TestingClass);
+        instance = new classWithColor();
+    });
+
+    it('should add a color property to an existing class', () => {
```

Comment:
I think this test can be removed

Reply:
I disagree. I added this to ensure that the mixin actually sets a class on the consuming class in order to ensure that it meets the interface. Mixins seem really janky to me, and this test makes me feel comfortable that the mixin is doing what I expect.

Reply:
But you're not testing for a class here. That's in the following test. Here, you're just setting a property directly on the instance, then checking the instance to see if that property is still set. Or are you testing that setting a color that exists as an possible color choice on that instance should set that color on the instance? To cover the inverse scenario of throwing an error if the color is not one of the choices on the instance? If that's the case, I think the test needs reworded to something like: `should be able to set a color defined by the mixin instance`.

Reply:
@lderrickable I've resolved this in c4e7c2839bfbc887537b1b197107401c9f9fea7a